### PR TITLE
fix: correct formatting and add some comments

### PR DESCRIPTION
**Pull request by l3gacyb3ta**
Haiii ishan

### CODE REVIEW

File: client/duckcoin.go [hunk 1]
```diff
@@ -44,6 +45,7 @@ Examples:
    duckcoin 1 -t nSvl+K7RauJ5IagU+ID/slhDoR+435+NSLHOXzFBRmo= -a 3 -m "Payment of 3 Quacks to Ishan"`
 )
 
+// A Block is a data structure that represents a block in the blockchain.
```
Comment (quackduck) [inline client/duckcoin.go]: IMO this doesn’t explain much about what blocks are, I can see what you’re trying to do tho.

I made Duckcoin to learn about how crypto works, and there’s a real lack of simple implementations, so I promised I would document the Duckcoin code really well, thanks for helping me!
```suggestion
// A Block is a data structure that represents a validated set of transactions with proof of work, which makes it really hard to rewrite the blockchain.
```

File: client/duckcoin.go [hunk 2]
```diff
@@ -34,6 +34,7 @@ var (
 	pubkeyFile  = configDir + "/pubkey.pem"
 	privkeyFile = configDir + "/privkey.pem"
 	urlFile     = configDir + "/url.txt"
+	// Difficulty is how hard it is to mine
```
Comment (quackduck) [inline client/duckcoin.go]:
```suggestion
	// Difficulty is how many zeros are needed in front of a block hash to be considered a valid block. Thus, this controls how much work miners have to do.
```

**Comment by l3gacyb3ta** [comment]
Seems great!

**Comment by quackduck** [comment]
Awesome, thank you so much for contributing to Duckcoin. 

Quack!

**Pull request merged**